input/pagecontent/general-guidance.md

```diff
@@ -110,6 +110,295 @@ Example: Patient resource with interpreter required and language is known
 }
 ~~~
 
+### Representing body site and laterality
+When using some AU Core profiles it may be desirable to represent relevant body site and laterality information associated with a record using CodeableConcept elements, notably:
```

Comment:
The phrase throughout "body site and laterality" needs to be adjusted. It implies that there are two completely distinct concepts. Possibly replace with "body site, which may include laterality,"

apply the rewording in similar introductory sentences.